### ISSUE TITLE
Blockscout throwing 500 when loading some tokens

### ISSUE THREAD

**Issue by markcutajar** [blockscout/blockscout#5857]
There are some issues with possibly the current release of Blockscout (started seeing this on 08/08/2022).

The following ERC721s throw a **500** with no information:
https://blockscout.com/xdai/mainnet/token/0x5550f0d022f706d03ad25a72c477684d3416193f/instance/9665/token-transfers
https://blockscout.com/xdai/mainnet/token/0x2043f330339e21afe6dc6207500b24acf74068fc/instance/210/token-transfers

However, this is not applicable to all tokens, some like the following works:
https://blockscout.com/xdai/mainnet/token/0x22c1f6050e56d2876009903609a2cc3fef83b415/instance/5467516/token-transfers

Image of issue:

![image](https://user-images.githubusercontent.com/24281545/183617543-f0cb818b-4c50-43a0-ab3a-6a2f7d37cad1.png)

**Comment by vbaranov** [comment]
@nikitosing error trace from the link above

```
Request: GET /token/0x5550f0d022f706d03ad25a72c477684d3416193f/instance/9665/token-transfers
** (exit) an exception was raised:
    ** (ErlangError) Erlang error: :enoent
        (elixir 1.13.1) lib/system.ex:1044: System.cmd("file", ["--mime-type", "/tmp/briefly-1660/briefly-39258-116725-4"], [])
        (file_info 0.0.4) lib/file_info.ex:16: FileInfo.get_info/1
        (block_scout_web 0.0.1) lib/block_scout_web/views/tokens/instance/overview_view.ex:67: BlockScoutWeb.Tokens.Instance.OverviewView.media_type/1
        (block_scout_web 0.0.1) lib/block_scout_web/templates/tokens/instance/overview/_details.html.eex:65: BlockScoutWeb.Tokens.Instance.OverviewView."_details.html"/1
        (block_scout_web 0.0.1) lib/block_scout_web/templates/tokens/instance/transfer/index.html.eex:2: BlockScoutWeb.Tokens.Instance.TransferView."index.html"/1
        (phoenix 1.5.13) lib/phoenix/view.ex:310: Phoenix.View.render_within/3
        (phoenix 1.5.13) lib/phoenix/view.ex:472: Phoenix.View.render_to_iodata/3
        (phoenix 1.5.13) lib/phoenix/controller.ex:777: Phoenix.Controller.render_and_send/4
```

**Comment by feld** [comment]
> @nikitosing error trace from the link above
> 
> ```
> Request: GET /token/0x5550f0d022f706d03ad25a72c477684d3416193f/instance/9665/token-transfers
> ** (exit) an exception was raised:
>     ** (ErlangError) Erlang error: :enoent
>         (elixir 1.13.1) lib/system.ex:1044: System.cmd("file", ["--mime-type", "/tmp/briefly-1660/briefly-39258-116725-4"], [])
>         (file_info 0.0.4) lib/file_info.ex:16: FileInfo.get_info/1
>         (block_scout_web 0.0.1) lib/block_scout_web/views/tokens/instance/overview_view.ex:67: BlockScoutWeb.Tokens.Instance.OverviewView.media_type/1
>         (block_scout_web 0.0.1) lib/block_scout_web/templates/tokens/instance/overview/_details.html.eex:65: BlockScoutWeb.Tokens.Instance.OverviewView."_details.html"/1
>         (block_scout_web 0.0.1) lib/block_scout_web/templates/tokens/instance/transfer/index.html.eex:2: BlockScoutWeb.Tokens.Instance.TransferView."index.html"/1
>         (phoenix 1.5.13) lib/phoenix/view.ex:310: Phoenix.View.render_within/3
>         (phoenix 1.5.13) lib/phoenix/view.ex:472: Phoenix.View.render_to_iodata/3
>         (phoenix 1.5.13) lib/phoenix/controller.ex:777: Phoenix.Controller.render_and_send/4
> ```

this is fixed by #6077
